### PR TITLE
Update using scarlet v0.5

### PR DESCRIPTION
**Pull request by fred3m**

### CODE REVIEW

**Comment by kadrlica** [comment]
@fred3m is scarlet built into newer versions of the stack? Can we use that rather than the standalone import?

**Comment by kadrlica** [comment]
Sorry for the noise above. It looks like somehow the kernel in your notebook was changed to `Python 3` when it should be `LSST`. I'll change and commit.

File: Deblending/scarlet_tutorial.ipynb [hunk 1]
```diff
@@ -442,29 +445,30 @@
...
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "LSST",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "lsst"
+   "name": "python3"
```
Comment (kadrlica) [inline Deblending/scarlet_tutorial.ipynb]: Needs to be the LSST kernel for LSST imports.

**Comment by fred3m** [comment]
@kadrlica that is correct, this notebook will probably not run on v18.1.1 since that version of the stack scarlet had a very different API than the current version.

**Pull request merged**